packages/react/src/recipes/FlexTemplate/FlexSection/FlexSection.tsx

```diff
@@ -0,0 +1,1011 @@
+                    <Image alt="Shopify" src={shopifyLogo} />
+                    <Image alt="Pinterest" src={pinterestLogo} />
+                    <Image alt="Twilio" src={twilioLogo} />
+                  </LogoSuite.Logobar>
+                </LogoSuite>
+              </Grid.Column>
+            </Grid>
+          )}
+
+          {cards && (
+            <Grid className={styles.normalizeMargin}>
+              <Grid.Column
+                span={{
+                  xsmall: 12,
+                  small: 12,
+                  medium: 6,
+                  large: 4,
+                  xlarge: 4,
+                  xxlarge: 4,
+                }}
+              >
+                <Box className={styles.heightFull}>
+                  <Card href="#" hasBorder={false} fullWidth>
+                    <Card.Image src={placeholderImage} alt="Placeholder card image" aspectRatio="16:9" />
+                    <Card.Heading>Card Title One</Card.Heading>
+                    <Card.Description>
+                      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt.
+                    </Card.Description>
+                  </Card>
+                </Box>
+              </Grid.Column>
+              <Grid.Column
+                span={{
+                  xsmall: 12,
+                  small: 12,
+                  medium: 6,
+                  large: 4,
+                  xlarge: 4,
+                  xxlarge: 4,
+                }}
+              >
+                <Box className={styles.heightFull}>
+                  <Card href="#" hasBorder={false} fullWidth>
+                    <Card.Image src={placeholderImage} alt="Placeholder card image" aspectRatio="16:9" />
+                    <Card.Heading>Card Title Two</Card.Heading>
+                    <Card.Description>
+                      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
+                    </Card.Description>
+                  </Card>
+                </Box>
+              </Grid.Column>
+              <Grid.Column
+                span={{
+                  xsmall: 12,
+                  small: 12,
+                  medium: 6,
+                  large: 4,
+                  xlarge: 4,
+                  xxlarge: 4,
+                }}
+              >
+                <Box className={styles.heightFull}>
+                  <Card href="#" hasBorder={false} fullWidth>
+                    <Card.Image src={placeholderImage} alt="Placeholder card image" aspectRatio="16:9" />
+                    <Card.Heading>Card Title Three</Card.Heading>
+                    <Card.Description>
+                      Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore.
+                    </Card.Description>
+                  </Card>
+                </Box>
+              </Grid.Column>
+            </Grid>
+          )}
+
+          {featuredBento && (
```

Comment:
Is this supposed to be so narrow? 

<img width="1794" height="976" alt="Screenshot 2025-10-02 at 11 30 00" src="https://github.com/user-attachments/assets/df26c32a-59da-4b93-b70d-38359ff84e57" />



Dotcom storybook for featured bento looks like like this, so wondering if we could match that? Or am I looking at a different component?

<img width="1778" height="667" alt="Screenshot 2025-10-02 at 11 30 17" src="https://github.com/user-attachments/assets/357780c4-1cd4-4d18-902b-547dba3d0b1b" />

Reply:
Thanks! Took a while but I got to the bottom of it 😄